docs/external_cmps.md

```diff
@@ -0,0 +1,81 @@
+# External components
+
+By default, plain ```docker-compose up```, regardless of the
+pipeline will start containers for following services:
+
+* MySQL
+* Keystone
+* ElasticSearch
+* Zookeeper
+* Kafka
+
+However each of them is treated, in the same time, as the
+dependency for *Monasca* hence can be externalized with the help
+of appropriate *environmental variables*.
+
+Following guide presents how to achieve that.
+
+## Using external OpenStack Keystone
+
+By default `monasca-docker` is using Keystone stored in separate
```

Comment:
**uses** Keystone stored in **a** separate

Reply:
Done